### ISSUE TITLE
Extend ENDD training script with VGG optimizations.

### ISSUE THREAD

**Issue by lennelov** [lennelov/endd-reproduce#50]
Dataset augmentation and cyclical learning rate should be added.
